docs/WindowsNative.md

```diff
@@ -36,6 +36,9 @@ then run
 
 ```bat
 npm install
+npm install webpack -g
```

Comment:
@okkenator can you comment on why these three npm commands were added?  I have just installed node on windows to test this and found these were not required.  I can run the following commands and be up and running without issues.

```
git clone https://github.com/compiler-explorer/compiler-explorer.git
cd compiler-explorer
npm install
npm start
```

I don't think we should be suggesting global installation of webpack in our docs unless there is a very good reason.

Reply:
Yes, I was not able to get it to work without that so I thought everyone else would need the same.  As I am new to using npm, it took me a little while to figure out.  I do not remember the exact error I was getting.  Maybe it should be listed as an option only if people have problems, or you could remove it as it is not ce specific but rather npm.